Sanity/pkcs11/basic/Makefile

```diff
@@ -0,0 +1,64 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think that you can remove Makefile, for now we are not using them anymore and slowly cleaning our repositories.

Reply:
Removed